### ISSUE TITLE
[WAITING ON CHECKLIST] Send money - Amount sent is copied as NaN in send money report page

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#29441]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.83-1

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to + > Send money
3. Send money to any user
4. Click on the IOU preview to open report page
5. Click on the avatar on the header
6. Right click on the IOU card and select Copy to clipboard
7. Paste it in the composer

## Expected Result:
The amount sent is copied correctly

## Actual Result:
The amount sent is copied as NaN

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

 

https://github.com/Expensify/App/assets/78819774/504424bc-fc69-48b5-b818-b6ed4176995d

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01b326fbae821e5272</li>
        <li>Upwork Job ID: 1712524952357588992</li>
        <li>Last Price Increase: 2023-10-12</li>
<li>Automatic offers: </li>
<ul>
<li>cubuspl42 | Reviewer | 27446145</li>
<li>saranshbalyan-1234 | Contributor | 27446146</li>
</ul></li>
    </ul>
</details>

**Comment by mountiny** [comment]
I dont think this has to be a deploy blocker

**Comment by jliexpensify** [comment]
Bump @cubuspl42 for reviews, cheers!

**Comment by cubuspl42** [comment]
We should go with @saranshbalyan-1234, I checked the code, and there's nothing much to discuss here

C+ reviewed 🎀 👀 🎀

**Comment by cubuspl42** [comment]
@saranshbalyan-1234 It was changed before your proposal was posted:

https://github.com/Expensify/App/issues/29441#issuecomment-1760085388

But prices can be adjusted at Expensify's discretion, in general.

**Comment by jliexpensify** [comment]
@saranshbalyan-1234 it looks like @mountiny manually changed this to $125 so that's what we'd be paying. Thanks!

**Comment by mountiny** [comment]
I ahve changed this to lower price as its lower priority and super easy fix too

**Comment by jliexpensify** [comment]
@cubuspl42 have we decided on a proposal?

**Comment by cubuspl42** [comment]
> @cubuspl42 have we decided on a proposal?

[Yes, we have](https://github.com/Expensify/App/issues/29441#issuecomment-1766029503)

**Comment by cubuspl42** [comment]
@jasperhuangg 

Would you take a look?

**Comment by jliexpensify** [comment]
Bump @jasperhuangg to [review](https://github.com/Expensify/App/issues/29441#issuecomment-1774555546)!

**Comment by jasperhuangg** [comment]
apologies for the delay, @saranshbalyan-1234's proposal makes perfect sense to me

**Comment by jliexpensify** [comment]
Sorry @saranshbalyan-1234 it looks like @mountiny specifically changed the price [here](https://github.com/Expensify/App/issues/29441#issuecomment-1768396992).

However, since this job was created before the recent pricing updates, I think it would be subject to an Urgency Bonus.

**Comment by jliexpensify** [comment]
@saranshbalyan-1234 @cubuspl42 [do we have a regression](https://github.com/Expensify/App/issues/30796)?

**Comment by cubuspl42** [comment]
@jliexpensify It looks like we do. I'm catching up.

**Comment by greg-schroeder** [comment]
fyi: https://github.com/Expensify/App/issues/30796#issuecomment-1808271436

**Comment by jliexpensify** [comment]
Thanks Greg - so to summarise: no penalty for regression here?

If so, Payment Summary:

- Contributor: @saranshbalyan-1234 $125
- C+: @cubuspl42 $125

[Upworks job here](https://www.upwork.com/jobs/~01b326fbae821e5272)

Will pay today

**Comment by jliexpensify** [comment]
Paid and job closed. @cubuspl42 please complete the checklist!

**Comment by cubuspl42** [comment]
- The PR that introduced the bug has been identified. Link to the PR:
    - https://github.com/Expensify/App/pull/27889
- The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
    - https://github.com/Expensify/App/pull/27889/files#r1392452654
- A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
    - No need for additional discussion
- Determine if we should create a regression test for this bug.
    - Up to the QA team
- If we decide to create a regression test for the bug, please propose the regression test steps to ensure the same bug will not reach production again.
    - Press to the "+" button, then select "Send money"
    - Proceed to send money to any user
    - Click on the IOU preview to access the report page
    - Click on the avatar located in the header
    - Right-click on the IOU card and choose "Copy to clipboard"
    - Paste the copied content into the composer
    - Verify that the pasted text correctly displays the user's name and amount
